pkg/runner/runner.go

```
@@ -543,6 +534,7 @@ func (r *Runner) RunEnumeration(pctx context.Context) error {
 						gologger.Warning().Msgf("Couldn't retrieve http response for %s: %s\n", ip, err)
 						return
 					}
+					defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)